add options to mount read-only and set the fuse-mount-option allow_other

RubenKelevra opened the pull request:
fixes #13

SupraSummus commented on bin/ipfs-api-mount:
```diff
@@ -18,6 +18,8 @@ if __name__ == '__main__':
     parser.add_argument('-b', '--background', action='store_true', help='Run in background.')
     parser.add_argument('-m', '--multithreaded', action='store_true', help='Use multiple threads to handle FS requests.')
+    parser.add_argument('--allow-other', action='store_true', help='Set fuse mount option \'allow_other\'')
+    parser.add_argument('--read-only', action='store_true', help='Set \'ro\' mount option instead of \'rw\'')
```
I feel like `ro` should be always set, and theres no need to expose it as a parameter.

SupraSummus commented:
Thanks for contribution.

I've rebased and merged it locally. Also, I've removed the read-only parameter. Now the mount is always read-only.

RubenKelevra commented:
> Thanks for contribution.
> 
> I've rebased and merged it locally. Also, I've removed the read-only parameter. Now the mount is always read-only.

I wasn't sure about any plans to do write requests in the future, that's the reason for exposing it. :)

SupraSummus commented:
I thought of implementing write-enabled mount point, indeed. But I'm not going to write this any time soon. PR's are welcome ;D
